utils/remove_node_configuration.yml

```diff
@@ -43,21 +39,24 @@
   roles:
     - { role: remove_node/remove_slurm_node } # noqa: role-name[path]
 
-- name: Proceeding to remove slurm node
-  tags: slurm_node
-  hosts: localhost
-  connection: ssh
-  gather_facts: false
-  roles:
-    - { role: remove_node/validate_slurm_node } # noqa: role-name[path]
-
 - name: Remove slurm node
   tags: slurm_node
-  hosts: remove_slurm_node
+  hosts: slurm_control_node, slurm_node
   connection: ssh
   gather_facts: true
-  roles:
-    - { role: remove_node/remove_slurm_node } # noqa: role-name[path]
+  # roles:
+    # - { role: remove_node/remove_slurm_node } # noqa: role-name[path]
+  pre_tasks:
+    - name: Include input project directory
+      ansible.builtin.import_role:
+        name: ../utils/roles/include_input_vars
+      run_once: true
+      delegate_to: localhost
+  tasks:
+    - name: Remove slurm node
+      ansible.builtin.import_role:
+        name: scheduler/roles/slurm
+        tasks_from: rm_node
```

Comment:
can we give file name as remove_node and keep in remove node configuration only?